deploy/kubernetes/dolphinscheduler/values.yaml

```diff
@@ -621,18 +627,28 @@ worker:
       # -- `PersistentVolumeClaim` size
       storage: "20Gi"
   env:
-    # -- Worker max cpu load avg, only higher than the system cpu load average, worker server can be dispatched tasks
-    WORKER_MAX_CPU_LOAD_AVG: "1"
-    # -- Worker reserved memory, only lower than system available memory, worker server can be dispatched tasks, the unit is G
-    WORKER_RESERVED_MEMORY: "0.3"
-    # -- Worker execute thread number to limit task instances
+    # If set true, will open worker overload protection
+    WORKER_SERVER_LOAD_PROTECTION_ENABLED: false
+    # -- Worker max cpu usage, when the worker's cpu usage is smaller then this value, worker server can be dispatched tasks.
+    WORKER_SERVER_LOAD_PROTECTION_MAX_CPU_USAGE_PERCENTAGE_THRESHOLDS: 0.7
+    # -- Worker max jvm memory usage , when the worker's jvm memory usage is smaller then this value, worker server can be dispatched tasks.
+    WORKER_SERVER_LOAD_PROTECTION_MAX_JVM_MEMORY_USAGE_PERCENTAGE_THRESHOLDS: 0.7
+    # -- Worker max memory usage , when the worker's memory usage is smaller then this value, worker server can be dispatched tasks.
+    WORKER_SERVER_LOAD_PROTECTION_MAX_SYSTEM_MEMORY_USAGE_PERCENTAGE_THRESHOLDS: 0.7
+    # -- Worker max disk usage , when the worker's disk usage is smaller then this value, worker server can be dispatched tasks.
+    WORKER_SERVER_LOAD_PROTECTION_MAX_DISK_USAGE_PERCENTAGE_THRESHOLDS: 0.7
```

Comment:
A comment for `WORKER_EXEC_THREADS` was deleted by mistake
```
# -- Worker execute thread number to limit task instances
```

Reply:
Done